di_website/home/templatetags/navigation_tags.py

```diff
@@ -60,3 +62,17 @@ def secondary_menu(context, parent, calling_page=None):
         # required by the pageurl tag that we want to use within this template
         'request': context.get('request'),
     }
+
+
+@register.inclusion_tag('tags/navigation/breadcrumbs.html', takes_context=True)
+def breadcrumbs(context):
+    self = context.get('self')
+    if self is None or self.depth <= 2:
+        # When on the home page, displaying breadcrumbs is irrelevant.
+        ancestors = ()
+    else:
+        ancestors = Page.objects.ancestor_of(self, inclusive=True).filter(depth__gt=1)
```

Comment:
I don't have any intuition on how draft/live parent/child relationships work (can a child be live and a parent be draft?) Could you please change to `ancestors = Page.objects.live().ancestor_of(self, inclusive=True).filter(depth__gt=1)` just in case?

Reply:
Ahh, sorry... I always forget this... thanks.

Reply:
Done!